packages/engine/src/output/buffer/util.rs

```diff
@@ -1,42 +1,74 @@
-use std::path::PathBuf;
-
 use memory::shared_memory::MemoryId;
 
-use crate::{
-    output::{buffer::RELATIVE_PARTS_FOLDER, error::Result},
-    proto::ExperimentId,
-};
+use crate::{output::buffer::RELATIVE_PARTS_FOLDER, proto::ExperimentId};
+
+// TODO: move this to top-level
+#[derive(PartialEq, Eq, Clone)]
+pub enum EngineExitStatus {
```

Comment:
Why is this in this file?

Reply:
Because `cleanup_experiment` is here.

Reply:
I'm not sure that reasoning makes sense though. If we look at what this module actually is
```output:buffer::util```

This enum has literally nothing to do with how we buffer simulation output.